instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py

```diff
@@ -177,10 +177,8 @@ def test_suppress_instrumentation(self):
     def test_not_recording(self):
         with mock.patch("opentelemetry.trace.INVALID_SPAN") as mock_span:
             RequestsInstrumentor().uninstrument()
-            # original_tracer_provider returns a default tracer provider, which
-            # in turn will return an INVALID_SPAN, which is always not recording
             RequestsInstrumentor().instrument(
-                tracer_provider=self.original_tracer_provider
+                tracer_provider=trace._DefaultTracerProvider
```

Comment:
Looks like the intention was to ensure that the instrumentation works with noop tracers.

Reply:
Where is `trace.__DefaultTracerProvider`?

Reply:
Also should probably remove `original_tracer_provider` from [TestBase](https://github.com/open-telemetry/opentelemetry-python/blob/main/tests/util/src/opentelemetry/test/test_base.py#L29)

Reply:
It is not directly used but I think might still be useful as it remembers the registered provider before a test case runs and automatically restores it back after a test case finishes running.